components/create/WorkHistory.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import CreatePortfolioContext from 'components/create/CreatePorfolioContext';
+import { Stack } from '@mui/material';
+import { useContext, useRef, RefObject } from 'react';
+import { StyledTextField, StyledButton } from 'components/create/Styled';
+const WorkHistory = () => {
+  const formRef = useRef() as RefObject<HTMLFormElement>;
+  const entry = {
+    position: '',
+    company: '',
+    dateWorked: '',
+    description: '',
+  };
```

Comment:
better to use useState instead of regular variables, from Slack:

> State variables is more idiomatic for React, enables re-rendering when the value changes, and gives you more control over the values that are stored and displayed to the user

Reply:
fixed